app/src/main/java/com/batch/mcs/finalproject/ChatFragment.java

```diff
@@ -0,0 +1,108 @@
+package com.batch.mcs.finalproject;
+
+import android.content.Context;
+import android.net.Uri;
+import android.os.Bundle;
+import android.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+
+/**
+ * A simple {@link Fragment} subclass.
+ * Activities that contain this fragment must implement the
+ * {@link ChatFragment.OnFragmentInteractionListener} interface
+ * to handle interaction events.
+ * Use the {@link ChatFragment#newInstance} factory method to
+ * create an instance of this fragment.
+ */
+public class ChatFragment extends Fragment {
+    // TODO: Rename parameter arguments, choose names that match
```

Comment:
Nop.... please erase all default code that should not be injected on the first place... any questions ask Carlos or Joe